src/components/Charts/ChartPage.tsx

```diff
@@ -0,0 +1,265 @@
+import React, { useState, useEffect } from "react";
+import { BarChart, Bar, XAxis, YAxis, CartesianGrid } from "recharts";
+import IconButton from "@mui/material/IconButton";
+import Box from "@mui/material/Box";
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import { Link } from "react-router-dom";
+import { getMeetingsData } from "../../api/meetings";
+import dayjs from "dayjs";
+import COLORS from "../../constants/CustomColors";
+import weekOfYear from "dayjs/plugin/weekOfYear";
+import weekYear from "dayjs/plugin/weekYear";
+import Chart from "./Chart";
+import MenuItem from "@mui/material/MenuItem";
+import Select, { SelectChangeEvent } from "@mui/material/Select";
+import { Typography } from "@mui/material";
+dayjs.extend(weekOfYear);
+dayjs.extend(weekYear);
+
+let fillColor = COLORS.ADMINCOLOR;
+
+const ChartPage = () => {
+    const months = [
+        "January",
+        "February",
+        "March",
+        "April",
+        "May",
+        "June",
+        "July",
+        "August",
+        "September",
+        "October",
+        "November",
+        "December",
+    ];
+
+    const monthsArray = months.map((month) => ({ name: month, value: 0 }));
+    const daysOfWeek = [
+        "Sunday",
+        "Monday",
+        "Tuesday",
+        "Wednesday",
+        "Thursday",
+        "Friday",
+        "Saturday",
+    ];
+    const daysArray = daysOfWeek.map((day) => ({ name: day, value: 0 }));
+    const [selectedYear, setSelectedYear] = useState(dayjs().year());
```

Comment:
Why do you have 3 states for year, month and day and not one for the whole date?